docker/start_isolate.sh

```diff
@@ -59,4 +66,7 @@ else
     print "Skipping isolate-check-environment"
 fi
 
+print "Executing $@"
+exec "$@"
```

Comment:
According to https://linux.die.net/man/1/sh#:~:text=exec%20%5B%2Dcl%5D%20%5B%2Da%20name%5D%20%5Bcommand%20%5Barguments%5D%5D, this does nothing if $@ is empty and execution will continue. Am I understanding this correctly? Thanks. @thepluck